stock_split_picking/wizards/stock_split_picking.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Hunki Enterprises BV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockSplitPicking(models.TransientModel):
+    _name = "stock.split.picking"
+    _description = "Split a picking"
+
+    mode = fields.Selection(
+        [
+            ("quantity", "Quantities"),
+            ("move", "One picking per move"),
+            ("selection", "Select move lines to split off"),
+        ],
+        required=True,
+        default="quantity",
+    )
+
+    picking_ids = fields.Many2many(
+        "stock.picking",
+        default=lambda self: self._default_picking_ids(),
+    )
+    move_ids = fields.Many2many("stock.move")
+
+    def _default_picking_ids(self):
+        return self.env["stock.picking"].browse(self.env.context.get("active_ids", []))
+
+    def action_apply(self):
+        return getattr(self, "_apply_%s" % self[:1].mode)()
```

Comment:
stock_split_picking/wizards/stock_split_picking.py:31:30: UP031 Use format specifiers instead of percent format